src/traces/table/plot.js

```diff
@@ -0,0 +1,892 @@
+            var hasWrapSplitCharacter;
+            var hwsc = function(prefixSuffixedText) {return prefixSuffixedText.indexOf(c.wrapSplitCharacter) !== -1;};
+            d.wrappingNeeded = !d.wrapped && !userBrokenText && !latex && (hasWrapSplitCharacter = hwsc(prefixSuffixedText));
+            d.cellHeightMayIncrease = userBrokenText2 || latex || d.mayHaveMarkup || (hasWrapSplitCharacter === void(0) ? hwsc(prefixSuffixedText) : hasWrapSplitCharacter);
+            var textToRender;
+            if(d.wrappingNeeded) {
+                var hrefPreservedText = c.wrapSplitCharacter === ' ' ? prefixSuffixedText.replace(/<a href=/ig, '<a_href=') : prefixSuffixedText;
+                var fragments = hrefPreservedText.split(c.wrapSplitCharacter);
+                var hrefRestoredFragments = c.wrapSplitCharacter === ' ' ? fragments.map(function(frag) {return frag.replace(/<a_href=/ig, '<a href=');}) : fragments;
+                d.fragments = hrefRestoredFragments.map(function(f) {return {text: f, width: null};});
+                d.fragments.push({fragment: c.wrapSpacer, width: null});
+                textToRender = hrefRestoredFragments.join(c.lineBreaker) + c.lineBreaker + c.wrapSpacer;
+            } else {
+                delete d.fragments;
+                textToRender = prefixSuffixedText;
+            }
+            return textToRender;
+        })
+        .attr('alignment-baseline', function(d) {
+            return d.cellHeightMayIncrease ? null : 'hanging';
+        })
+        .each(function(d) {
+
+            var element = this;
+            var selection = d3.select(element);
+
+            // finalize what's in the DOM
+
+            var renderCallback = d.wrappingNeeded ? wrapTextMaker : updateYPositionMaker;
+            if(d.mayHaveMarkup || d.wrappingNeeded || d.latex) {
+                svgUtil.convertToTspans(selection, gd, renderCallback(allColumnBlock, element, tableControlView, gd, d));
+            } else {
+                d3.select(element.parentNode)
+                    // basic cell adjustment - compliance with `cellPad`
+                    .attr('transform', function(d) {return 'translate(' + xPosition(d) + ' ' + c.cellPad + ')';})
+                    .attr('text-anchor', function(d) {
+                        return ({
+                            left: 'start',
+                            center: 'middle',
+                            right: 'end'
+                        })[d.align];
+                    });
+            }
+        });
+}
+
+function isLatex(content) {
+    return typeof content === 'string' && content.match(c.latexCheck);
+}
+
+function columnMoved(gd, calcdata, i, indices) {
+    var o = calcdata[i][0].gdColumnsOriginalOrder;
+    calcdata[i][0].gdColumns.sort(function(a, b) {
+        return indices[o.indexOf(a)] - indices[o.indexOf(b)];
+    });
+
+    calcdata[i][0].columnorder = indices;
+
+    gd.emit('plotly_restyle');
```

Comment:
... with corresponding event data.

Reply:
... as reordering columns (I hope) can be done via `Plotly.restyle` too

Reply:
Yes, although I haven't yet tested it and maybe it'll need an `.exit().remove()` to `yColumn` if not done yet, so let me check.